### ISSUE TITLE
Improve Android 12+ splash images 

### ISSUE THREAD

**Issue by mattleibow** [dotnet/maui#10604]
### Description

The current behavior is that the splash screen is clipped on Android 12+. Previously, Android would just display the image as-is and you would have to do all the work yourself.

In Android 12, things got smarter and now the image is clipped inside a circle and you could potentially have a different foreground/background layer like the adaptive icons. 

<img src="https://user-images.githubusercontent.com/1096616/195070999-3a9d9c5d-30d8-428d-833a-d11e3a8eb2f9.png" width="300px" />

> https://developer.android.com/develop/ui/views/launch/splash-screen

### Public API Changes

None.

### Intended Use-Case

We could leverage the same properties of `<MauiIcon>` and do splash screens better. We can use the same logic to make adaptive icons, but just in the drawable folder and not generate the xml. Then the splash screen uses that drawable pair to make a nicer icon.

**Comment by mattleibow** [comment]
Another area is to also look at not generating larger images than can be used. Prior to Android 12, the image was the full available space. In 12+ the image is "108dp" but also not really...

I found that if I tried to actually make 108dp (mdpi) images it scaled up. In fact, no matter what size image, it always looks blurry... I am wondering if the adaptive xml is loading smaller images and then scaling up because according to de docs (https://developer.android.com/develop/ui/views/launch/splash-screen#splash-screen-animate-reqs):

> The icon area must be 432 dp (in other words, four times 108 dp of the area of an unmasked adaptive icon).

But generating an image of 432dp results in a terrible image - either massive clip or super scaled.

**Comment by janseris** [comment]
If image is not square it is clipped and not zoomed to fit into the circle
E. g. wide image with company logo with 8 letters is clipped by the circle so that only middle 3 letters are visible on Android 12.

**Comment by mattleibow** [comment]
@janseris  what version of maui are you using? I think this was fixed in the version that went out with .NET 7 RC 1. The image is now shrunk to fit inside the circle.

**Comment by mattleibow** [comment]
@janseris  what version of maui are you using? I think this was fixed in the version that went out with .NET 7 RC 1. The image is now shrunk to fit inside the circle.

**Comment by janseris** [comment]
@mattleibow latest nonpreview. NET 6 which comes with VS 17.3.5 I think because we use it on production

**Comment by mattleibow** [comment]
In the case of Android 12+, not that I can see.  Android is locking down the stuff so maybe you can see something here?

https://developer.android.com/develop/ui/views/launch/splash-screen

**Comment by fgiacomelli** [comment]
Hello, I just tryed with VS 17.5.0 preview 2, because I read that the problem of the round image on Android >= 12 was solved, but on my test I continue to see my splash icon, that is a rectangle, filled and stretched in a circle.
I think it's related to the #9886, that is marked as solved.
I'm deploying on Android 12, target version net 7.0.

In which version can we expect it will be fixed?

**Comment by h0lg** [comment]
> We could leverage the same properties of `<MauiIcon>` and do splash screens better.

I agree - a setting to scale the icon when fitting it inside the circle on Android would be really helpful for square-ish icons.
